integration/env/env.go

```diff
@@ -71,13 +79,38 @@ func CircleSHA() string {
 }
 
 func ClusterID() string {
-	return fmt.Sprintf("ci-awsop-%s-%s", TestedVersion(), CircleSHA()[0:5])
+	var parts []string
+
+	parts = append(parts, "awsci")
```

Comment:
A comment here explaining the different parts would be very useful, especially the dir hash part.

Reply:
Added godoc to `ClusterID`.